packages/components/src/autocomplete/autocompleter-ui.js

```diff
@@ -55,41 +56,53 @@ export function getAutoCompleterUI( autocompleter ) {
 			return null;
 		}
 
-		return (
-			<Popover
-				focusOnMount={ false }
-				onClose={ onReset }
-				placement="top-start"
-				className="components-autocomplete__popover"
-				anchor={ popoverAnchor }
-				ref={ popoverRef }
+		const ListBox = ( { hide } ) => (
+			<div
+				id={ listBoxId }
+				role="listbox"
+				className="components-autocomplete__results"
+				style={ hide ? { display: 'none' } : {} }
```

Comment:
Can you use the `VisuallyHidden` component here instead? I think the `display:none;` is preventing reading.

Reply:
Changed it. Does it work now?

Reply:
@ellatrix Works fine. Are you planning on adding more changes or is this ready for code review?

Reply:
I think it's fine as it is, though I might try to remove the slot/fill. Also, is there any way we can create an e2e test for this?

Reply:
@alexstine I made a small change. Does it still work? So the only thing left would be figuring out a good e2e test so we don't get any regressions here.

Reply:
@ellatrix It still works. To this date, I have never seen accessible use of React portals. Cool you were able to make it work.

For the E2E test, something like this I am thinking.
- Wait for iFrame load.
- Add default block.
- Navigate to input field.
- Type /gr
- Query selector and check that the suggestions list appears in the iFrame DOM. I would do it like this.
```
const input = document.ownerDocument.activeElement.getAttribute( 'aria-owns' );
const iFrame = document.querySelector( 'iframe' );
const suggestions = document.getElementById( input );
const checkSuggestionsInIframe = iframe.contains( suggestions );
await expect( await checkSuggestionsInIframe ).toBe( true );
```

The idea, make sure the `aria-owns` attribute has a valid ID for an element located inside the iFrame. Hope it helps.

Reply:
Right, I was just hoping we would find a test that's more "end to end" instead of just checking IDs in the DOM. But Chrome and/or Playwright don't have a screen reader emulation I guess?

Reply:
I would be shocked if it did. Not sure if static checking could somehow look for this issue or not. Kind of doubt that as well.